Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -409,15 +409,15 @@ def process_alert(self, alert, hops=0):
 
         if alert_type == 'add_torrent_alert':
             infohash = str(handle.info_hash())
-            if infohash in self.torrents:
+            if infohash in self.torrents and not self.torrents[infohash][0].deferred_added.called:
```

Comment:
One note: if this Deferred can be callback-ed from another function, `.called` can be racy.